config_samples/examples/client.example.toml

```
@@ -0,0 +1,18 @@
+chain_id = "00000000-0000-0000-0000-000000000000"
+
+[account]
+id = "alice@wonderland"
+public_key = "ed01207233BFC89DCBD68C19FDE6CE6158225298EC1131B6A130D1AEB454C1AB5183C0"
+private_key.digest_function = "ed25519"
+private_key.payload = "9ac47abf59b356e0bd7dcbbbb4dec080e302156a48ca907e47cb6aea1d32719e7233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
+
+[api]
+torii_url = "http://127.0.0.1:8080/"
+basic_auth.login = "mad_hatter"
+basic_auth.password = "ilovetea"
+
+[transaction]
+time_to_live = 100_000
+status_timeout = 100_000
+add_nonce = false
```

```suggestion
nonce = false
```

might be equally appropriate